packages/now-build-utils/src/detectors/angular.ts

```diff
@@ -1,14 +1,18 @@
 import { DetectorParameters, DetectorResult } from '../types';
 
 export default async function detectAngular({
-  fs: { hasDependency },
+  fs: { hasDependency, getPackageJsonBuildCommand, getDependencyVersion },
 }: DetectorParameters): Promise<DetectorResult> {
   const hasAngular = await hasDependency('@angular/cli');
   if (!hasAngular) return false;
   return {
-    buildCommand: 'ng build',
-    buildDirectory: 'dist',
+    buildCommand: (await getPackageJsonBuildCommand()) || 'ng build',
```

Comment:
The parens around the `await` cal make this hard to read. What if we made the API look more like:

```suggestion
    buildCommand: await getPackageJsonScript('build,' 'ng build'),
```

Where if a `build` script is defined, then `npm/yarn run build` would be returned, otherwise the fallback of `ng build` would be used.